frontend/app/pages/dashboard/dashboard.tsx

```diff
@@ -87,7 +55,13 @@ export function DashboardPage() {
         </BreadcrumbList>
       </Breadcrumb>
       <h1 className="text-3xl font-bold mb-2">Dashboard</h1>
-      <BreakingNews breakingNews={breakingNews} />
+      <Alert
+        onClick={() => navigate("/dashboard/breaking")}
+        className="hover:bg-gray-100 hover:cursor-pointer mb-2"
+      >
+        <Megaphone />
+        <AlertTitle>Click here to view the latest breaking news!</AlertTitle>
+      </Alert>
```

Comment:
```suggestion
      
      <Link to="/dashboard/breaking">
        <Alert className="hover:bg-gray-100 mb-2">
          <Megaphone />
          <AlertTitle>Click here to view the latest breaking news!</AlertTitle>
        </Alert>
      </Link>
```

Please use Link component to navigate from html